Src/FluentAssertions/Equivalency/SelfReferenceEquivalencyAssertionOptions.cs

```diff
@@ -416,6 +416,14 @@ public TSelf Using(IMemberMatchingRule matchingRule)
             return AddMatchingRule(matchingRule);
         }
 
+        /// <summary>
+        /// Adds an ordering rule to the ones already added by default, and which is evaluated after all existing rules.
+        /// </summary>
+        public TSelf Using(IOrderingRule orderingRule)
```

Comment:
Should a custom ordering rule not override the other ordering rules?

Reply:
That would be inconsistent with
```cs
public TSelf WithStrictOrderingFor(Expression<Func<IMemberInfo, bool>> predicate)
{
    orderingRules.Add(new PredicateBasedOrderingRule(predicate));
    return (TSelf)this;
}
```

Reply:
and with a custom ordering rule we also wouldn't want to get rid of the strict byte array ordering rule.
I can also imagine having more than 1 ordering rule. e.g. for specific collection types you want to enforce strict ordering, and you could solve that with a rule per collection type

Reply:
Ah yeah, you're right. If there's a rule following `ByteArrayOrderingRule` that returns `OrderStrictness.NotStrict`, it will overrule the byte array ordering rule.